src/Blocks/components/checkbox/checkbox.php

```diff
@@ -39,21 +39,24 @@
 $checkboxFieldAttrs = Helpers::checkAttr('checkboxFieldAttrs', $attributes, $manifest);
 $checkboxIcon = Helpers::checkAttr('checkboxIcon', $attributes, $manifest);
 $checkboxIsHidden = Helpers::checkAttr('checkboxIsHidden', $attributes, $manifest);
+$checkboxTwSelectorsData = Helpers::checkAttr('checkboxTwSelectorsData', $attributes, $manifest);
 
 if ($checkboxAsToggle) {
 	$componentClass = "{$componentClass}-toggle";
 }
 
+$twClasses = FormsHelper::getTwSelectors($checkboxTwSelectorsData, ['checkbox']);
```

Comment:
```suggestion
$twClasses = FormsHelper::getTwSelectors($checkboxTwSelectorsData, ['checkbox']);
```
Can `getTwSelectors` accept a variable number of args, so it behaves like the other Tailwind functions from Libs?

Reply:
this it not so similar to the libs helpers as this helper will filter out top level keys that could be used in this component. The array allows this to use multiple top level key like in the field component.